CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## [Unreleased](https://github.com/Dynatrace/openkit-native/compare/v1.1.0...HEAD)

```

Comment:
We have one bullet point in the "Changed" list and 23 commits in the compare link.